atlas/packages/components/src/components/VideoPreview/VideoPreview.tsx

```diff
@@ -1,47 +1,86 @@
-import React from "react"
+import React from "react";
 
-import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles"
-import Avatar from "./../Avatar"
+import { makeStyles, VideoPreviewStyleProps } from "./VideoPreview.styles";
+import Avatar from "./../Avatar";
 
 type VideoPreviewProps = {
-  title: string
-  channel?: string
-  channelImg?: string
-  showChannel?: boolean
-  poster?: string
-  onClick?: any
-  onChannelClick?: any
-} & VideoPreviewStyleProps
+	title: string;
+	channel?: string;
+	channelImg?: string;
+	showChannel?: boolean;
+	showMeta?: boolean;
+	time?: string;
+	views?: string;
+	poster?: string;
+	onClick?: any;
```

Comment:
Wouldn't it be better to have at least very generic Function interface here? Because that's typed as `any` the TS compiler will skip checking for whether the value is already there. There's a bug currently in the code, because `onClick` is possibly `undefined` but we call it unconditionally. If we change the prop type from `any` then the compiler will give us an error

Reply:
@Gamaranto `onChannelClick` was changed but `onClick` is still `any`. I'm also wondering whether we need to pass the event down the tree or maybe an empty callback will be enough

Reply:
Fixed